ajax/libs/koto/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "koto",
+  "version": "0.3.0",
+  "description": "A d3 chart framework written in ES6",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kotojs/kotojs.git"
+  },
+  "author": "Nick Randall <nick.randall@domo.com>",
+  "homepage": "http://kotojs.org",
+  "license": "MIT",
+  "filename": "koto.js",
+  "keywords": [
+    "d3",
+    "chart",
+    "es6",
+    "visualization",
+    "koto"
+  ],
+  "autoupdate": {
```

Comment:
`koto` has also been published on NPM and there are more releases than GitHub. ref: https://www.npmjs.com/package/koto
Please use npm auto-update instead of git auto-update in this case.